x-pack/platform/plugins/shared/ml/public/application/components/ml_saved_objects_spaces_list/ml_saved_objects_spaces_list.tsx

```diff
@@ -33,11 +33,12 @@ const modelObjectNoun = i18n.translate('xpack.ml.management.jobsSpacesList.model
   defaultMessage: 'trained model',
 });
 
+const FALLBACK_SPACES_ID: string[] = [];
 export const MLSavedObjectsSpacesList: FC<Props> = ({
   spacesApi,
-  spaceIds,
+  spaceIds = FALLBACK_SPACES_ID,
   id,
-  mlSavedObjectType,
+  mlSavedObjectType = ML_JOB_SAVED_OBJECT_TYPE as MlSavedObjectType,
```

Comment:
This change has introduced a bug where it's not possible to assign jobs to spaces using the original ML stack management page.
`ML_JOB_SAVED_OBJECT_TYPE` cannot be used here as it is not of type `MlSavedObjectType`